beetsplug/bandcamp/__init__.py

```diff
@@ -218,9 +218,11 @@ def candidates(
 
         if album == "":
             search = {
-                "query": artist + " - " + item.title,
-                "artist": artist,
-                "label": label,
+                "query": item.artist + " - " + item.title
+                if item.artist != ""
+                else item.title,
+                "artist": item.artist,
```

Comment:
```suggestion
                "artist": artist,
```

Here, I think we want to be careful regarding how we use `artist` and `item.artist`.

* `item.artist` comes from the music file being imported
* `artist` comes from the search (the same one you enter in the import search prompt, after you press key **E**).

You can probably see that using `item.artist` here would means that your search input will is ignored.

Reply:
same here as with the label, my mistake 😅